### DISCUSSION TITLE
Key [permissions_cascade] doesn't exist

### DISCUSSION THREAD

**Discussion by danfoxx** [wheels-dev/wheels#1297] [General]
Hi everybody

I get a strange error after deploying a working environment to another hardware.

1. Zip-Filed working project folder
2. Exported MySQL-Dump 
3. Installed following software on new hardware (Windows 10)
  - MySQL
  - Java Runtime
  - commandbox
  - etc
 
At the end - i get the folllowing error as per screen shot

Any ideas? Help would be very much apreciated- Thx.

PS: Lucee sais, DB connection is working and the key is in the settings table set to "true"

![cfwheels_error](https://github.com/cfwheels/cfwheels/assets/3247617/1a71bf26-fbe4-4ce7-8625-055a82c20fc3)

**Comment by neokoenig** [comment]
Looks like `application.settings` isn't being set properly. Does a lucee / or application restart not sort it?

**Reply by danfoxx** [reply]
I restared many times. 
It occured once before  - when I copied the development environment and set the new one to "testing"  before I started the server for the first time. 
Then I changed settings back to "development" and everything worked thine.
At this time, I also had "Server Profile" set to "testing" which does not exist.
I also changed that, and the application was running again.

Sorry for the bumpy explanation

**Reply by danfoxx** [reply]
Thank you very much for your reply !!!
So much appreciated
